lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.upload_block_blob_from_file('<container name>', '<blob name>', '<file path>')
```

Comment:
Instead of directly calling request on service, we suggest using following format for resource creation. 

service.collection.create() i.e.
azure_storage_service.blobs.create(<argument-hash>)